doc/build_instructions/packages.md

```diff
@@ -0,0 +1,31 @@
+# Package Instructions
+## Linux Packages [ ![Download](https://api.bintray.com/packages/simonsan/openage-packages/openage-linux-releases/images/download.svg) ](https://bintray.com/simonsan/openage-packages/openage-linux-releases/_latestVersion)
+
+Currently we have only the packages available that you can see at [repology](https://repology.org/project/openage/versions).  We are working on *.deb-packaging and AppImages though, you can follow the progress at [bintray](https://bintray.com/simonsan/openage-packages/openage-linux-releases).
+
+## Windows Packages [ ![Release](https://api.bintray.com/packages/simonsan/openage-packages/openage-windows-release/images/download.svg) ](https://bintray.com/simonsan/openage-packages/openage-windows-release/_latestVersion)
+
+### Instructions
+1. Download and check [signature](https://gist.githubusercontent.com/simonsan/817cdc5ef5cd5f53875bc52bf4cd7a2f/raw/)
+2. Install
+3. Set a new environment variable *AGE2DIR* to the path of your Age2-directory. You could also add that in the *openage.bat*-file, if you like.
+
+    __NOTE:__ We currently support Age2-Versions until HD4.3 and old AoC 1.0c, if you have the last Steam-Version ***read below***. A new converter version comes soon.
+4. Double-click *openage.bat*
+5. Wait for the conversion of the assets
+6. Enjoy
+
+### How to set AoE2HD to Version 4.3
+1. Make a backup of your existing Age2HD-folder
+
+    __NOTE:__ Standard path is `C:\Program Files (x86)\Steam\steamapps\common\Age2HD`
+
+2. Open Steam library
+3. Right-click on "Age of Empires II: HD Edition" and click on `Properties`
+4. Go to `Betas`
+5. Choose `Patch43`
+6. Close the window and wait for the download to finish
+
+
+## Troubleshooting
+Please give ***feedback for potential problems*** in our [Matrix](https://riot.im/app/#/room/#sfttech:matrix.org) channel or on [reddit](https://www.reddit.com/r/openage/). For first-aid also consider taking a look into the [Troubleshooting](https://github.com/SFTtech/openage/blob/master/doc/troubleshooting.md) guide.
```

Comment:
> For first-aid ...

openage cannot heal wounds (yet), so I would leave this part of the sentence out :D

Also, the two sentences should be swapped. People should look into the troubleshooting guide first, then write us.